modules/xfem/test/tests/stateful_extraqp/tests

```diff
@@ -0,0 +1,30 @@
+[Tests]
+  design = 'XFEMMaterialManager.md'
+  issues = '#7291'
+  [manager]
+    type = Exodiff
+    exodiff = manager_out.e
+    input = manager.i
+    unique_id = true
+    allow_test_objects = true
+    requirement = 'The XFEM module shall provide a means of managing stateful materials on added XFEM quadrature points'
+  []
+  [manager_constraint]
+    type = Exodiff
+    exodiff = manager_constraint_out.e-s002
+    input = manager_constraint.i
+    unique_id = true
+    map = false
+    allow_test_objects = true
+    requirement = 'The XFEM module shall represent a propagating crack in a 2D mechanics problem with stateful materials'
```

Comment:
'with stateful materials on the added XFEM quadrature points'

Reply:
The suggestion feature is really useful for such md file changes...